### PR TITLE
Ft/redo basic section

### PR DESCRIPTION
**Pull request by Mr-emeka**

### CODE REVIEW

File: src/components/Resume/Header.js [hunk 1]
```diff
@@ -25,71 +25,86 @@ export const Header = ({
...
-      <ul data-testid="Address">
-        <li>{header.address}</li>
-        <li>{header.city}</li>
-        <li>{header.state}</li>
-        <li>{header.zip}</li>
-        <li>{header.country}</li>
+    <header className="resume-header" style={{ fontFamily: font }}>
+      <h1 style={{ fontFamily: font }}>{header.name}</h1>
+      <ul>
+        {console.log(header)}
```
Comment (aamay001) [inline src/components/Resume/Header.js:31]: Please remove this console.log from here.

**Review by aamay001 — CHANGES_REQUESTED** [review]
This is looking good. A minor change was requested. Thanks.

File: src/components/Resume/Header.js (hunk 1 above)
Comment (Mr-emeka) [reply src/components/Resume/Header.js:31]: Oops, I totally forgot about that I would remove it now.

**Comment by aamay001** [comment]
@Mr-emeka there is an issue with the CI pipeline that is preventing the tests from running successfully. I've contacted MS to get support on this. I will merge in your changes once the issue has been resolved and all tests pass successfully. In the meantime, I have added the label `hacktoberfest-accepted` to your PR. Thanks for your patience with this.

**Pull request closed without merging**